### PR TITLE
Update apps-win32-supersedence.md

### PR DESCRIPTION
**Pull request by msbemba**
made changes to the supersede section to avoid ambiguity

Fixes https://github.com/MicrosoftDocs/memdocs/issues/4351

### CODE REVIEW

File: memdocs/intune/apps/apps-win32-supersedence.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ A *superseding app* is an app that updates or replaces other apps. A *superseded
 Given that an app could have multiple superseded apps, it is possible for an app to update a set of apps while replacing another set of apps at the same time.
 
 > [!NOTE]
-> End-users won't be able to check whether a specific Win32 app supersedence operation is an update or replacement in the Company Portal. In addition, when multiple apps supersede an app with available targeting in the Company Portal, the superseded app's details page will navigate to the app page of the first superseding app that was set up. For example, if app A is superseded by app B and C, and app B is superseded by app A first, then app A's detail page in the Company Portal will navigate to App B.
+> End-users won't be able to check whether a specific Win32 app supersedence operation is an update or replacement in the Company Portal. In addition, when multiple apps supersede an app with available targeting in the Company Portal, the superseded app's details page will navigate to the app page of the first superseding app that was set up. For example, if app A is superseded by app B and C, and app B supersedes app A first, then app A's detail page in the Company Portal will navigate to App B.
```
Comment (JohanFreelancer9) [inline memdocs/intune/apps/apps-win32-supersedence.md:82]:
```suggestion
> End-users won't be able to check whether a specific Win32 app supersedence operation is an update or replacement in the Company Portal. In addition, when multiple apps supersede an app with available targeting in the Company Portal, the superseded app's details page will navigate to the app page of the first superseding app that was set up. For example, if app A is superseded by apps B and C, and app B supersedes app A first, then app A's detail page in the Company Portal will navigate to App B.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Erikre Please review and signoff this PR. Thanks

**Comment by Erikre** [comment]
#sign-off

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**